README.markdown

```diff
@@ -287,6 +287,8 @@ protoc --plugin=node_modules/ts-proto/protoc-gen-ts_proto ./batching.proto -I.
 
 - With `--ts_proto_opt=useDate=false`, fields of type `google.protobuf.Timestamp` will not be mapped to type `Date` in the generated types.
 
+- With `--ts_proto_opt=outputMetaTypings=true`, meta typings will be generated that can later be used in other code generators.
```

Comment:
What do you think about `outputSchema=true`? I.e. using the term "schema" instead of "meta typing". Maybe pedantically a "typing" in the TS sense typically isn't available at runtime, and what you're building actually is, hence a slight nudge away from the term "typing". If you really want to use the term "meta" we could do `outputMetadata=true`. Wdyt?